internal/pkg/cli/job_package.go

```diff
@@ -129,6 +153,46 @@ func (o *packageJobOpts) askEnvName() error {
 	return nil
 }
 
+func (o *packageJobOpts) newPackageCmd() (*packageSvcOpts, error) {
+	ws, err := workspace.New()
+	if err != nil {
+		return nil, fmt.Errorf("new workspace: %w", err)
+	}
+	store, err := config.NewStore()
+	if err != nil {
+		return nil, fmt.Errorf("connect to config store: %w", err)
+	}
+	p := sessions.NewProvider()
+	sess, err := p.Default()
+	if err != nil {
+		return nil, fmt.Errorf("retrieve default session: %w", err)
+	}
+	prompter := prompt.New()
+	packageCmd := &packageSvcOpts{
```

Comment:
nit: can we add labels to the elements to increase readability?
```go
...
stackWriter: os.Stdout,
paramsWriter: ioutil.Discard,
...
```

Reply:
Oh, hahaha yes. My IDE spoils me.